### PR TITLE
samples:gui:lvgl: fix the nRF52840 pins to run sample

### PR DESCRIPTION
**Pull request by chtavares**
this commit correct the #17460 issue

Signed-off-by: Christian Tavares <christian.tavares@ossystems.com.br>

### CODE REVIEW

File: samples/gui/lvgl/nrf52840_pca10056.overlay [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+ * Copyright (c) 2019 O.S.Systems
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 &spi1 {
 	status = "okay";
+	sck-pin = <47>;
+	mosi-pin = <45>;
+	miso-pin = <46>;
```
Comment (jfischer-phytec-iot) [inline samples/gui/lvgl/nrf52840_pca10056.overlay]: @chtavares592 Could you also fix `spi-max-frequency` and set it to `spi-max-frequency = <10000000>;`?

**Review by dbkinder — APPROVED** [review]
+1 for doc changes

**Review by vanwinkeljan — APPROVED** [review]

File: samples/gui/lvgl/nrf52840_pca10056.overlay (hunk 1 above)
Comment (vanwinkeljan) [reply samples/gui/lvgl/nrf52840_pca10056.overlay]: The `spi-max-frequency` is set according the ili datasheet and the nrf spi driver will use the closest supported frequency that is lower then the max frequency.

Comment (jfischer-phytec-iot) [reply samples/gui/lvgl/nrf52840_pca10056.overlay]: okay, then maybe `spi-max-frequency = <15000000>;`?  `15151515` looks like a typo.

Comment (chtavares592) [reply samples/gui/lvgl/nrf52840_pca10056.overlay]: I just kept the same value but I tested it with 10000000 and it works too. If is necessary I can include this change on that commit.

**Review by anangl — APPROVED** [review]

File: samples/gui/lvgl/nrf52840_pca10056.overlay (hunk 1 above)
Comment (ioannisg) [reply samples/gui/lvgl/nrf52840_pca10056.overlay]: @jfischer-phytec-iot , if you believe this is really a typo, then @chtavares592 could just update the value, IMO, as part of this PR.

Comment (vanwinkeljan) [reply samples/gui/lvgl/nrf52840_pca10056.overlay]: > 15151515 looks like a typo.

Sorry EE background, just converted the timing requirements and then you end up 15.15... MHz

Btw no problem with another value (eg 15000000) if it makes it more clear in the end the SPI driver will pick the actual speed

Comment (chtavares592) [reply samples/gui/lvgl/nrf52840_pca10056.overlay]: So, Am I need to change the `spi-max-frequency` value?

**Pull request merged**